pyatv/core/scan.py

```diff
@@ -392,9 +391,16 @@ async def _lookup_services_and_models(
     ) -> Tuple[Dict[str, List[AsyncServiceInfo]], Dict[str, str]]:
         """Lookup services and aggregate them by address."""
         infos = self._build_service_info_queries()
-        await asyncio.gather(
-            *[info.async_request(self.zeroconf, zc_timeout) for info in infos]
-        )
+        requests: List[Awaitable[bool]] = []
+        multicast_requests = [
+            info.async_request(self.zeroconf, zc_timeout) for info in infos
+        ]
```

Comment:
I think we can take this out if there are `hosts` in `self.hosts` but leaving it in for testing